elyra/templates/kubeflow/v2/python_dsl_template.jinja2

```diff
@@ -18,6 +18,8 @@ from kfp.dsl import PipelineTask
 from kfp.kubernetes import common
 from kfp.kubernetes import kubernetes_executor_config_pb2 as pb
 
+from kfp.kubernetes import add_toleration
```

Comment:
@harshad16 we will need patches for the tensorflow notebook.
Also, since now we have `1.2.0`, should we directly import the `add_pod_label` and `add_pod_annotation` methods?

Reply:
Yes, will update

Reply:
thank you.